src/include/homeobject/blob_manager.hpp

```diff
@@ -10,7 +10,7 @@
 namespace homeobject {
 
 ENUM(BlobError, uint16_t, UNKNOWN = 1, TIMEOUT, INVALID_ARG, NOT_LEADER, UNKNOWN_SHARD, UNKNOWN_BLOB, CHECKSUM_MISMATCH,
-     READ_FAILED, INDEX_ERROR, SEALED_SHARD);
+     READ_FAILED, INDEX_ERROR, SEALED_SHARD, TOMBSTONE_BLOB, ALIVE);
```

Comment:
I probably didn't get why TOMBSTONE_BLOB/ALIVE is an BlobError in the first place. 
After reading through the usage of these two error code, both of them can be categorized into INDEX_ERROR, since they all mean we queried some thing from the index with expected state, but index didn't return correctly, right?

Reply:
ack